### PR TITLE
Fix postgresql helm parameters during e2e test installation

### PR DESCRIPTION
**Pull request by shubham1172**
# Description

The existing overrides file https://github.com/dapr/dapr/blob/master/tests/config/postgres_override.yaml does not set the affinity and persistence correctly. See https://github.com/bitnami/charts/blob/main/bitnami/postgresql/README.md for correct values.

## Issue reference

Please reference the issue this PR will close: #6780

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/dapr_tests.mk [hunk 1]
```diff
@@ -532,7 +532,7 @@ delete-test-env-zipkin:
 	$(KUBECTL) delete -f ./tests/config/zipkin.yaml -n $(DAPR_TEST_NAMESPACE)
 
 # Setup the test environment by installing components
-setup-test-env: setup-test-env-kafka setup-test-env-redis setup-test-env-mongodb setup-test-env-postgres setup-test-env-k6 setup-test-env-zipkin setup-test-env-postgres
+setup-test-env: setup-test-env-kafka setup-test-env-redis setup-test-env-mongodb setup-test-env-postgres setup-test-env-k6 setup-test-env-zipkin
```
Comment (ItalyPaleAle) [inline tests/dapr_tests.mk:535]: Why was this changed?
```suggestion
setup-test-env: setup-test-env-kafka setup-test-env-redis setup-test-env-mongodb setup-test-env-postgres setup-test-env-k6 setup-test-env-zipkin setup-test-env-postgres
```

**Review by ItalyPaleAle — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

File: tests/dapr_tests.mk (hunk 1 above)
Comment (shubham1172) [reply tests/dapr_tests.mk]: It was a duplicate.
